packages/next-server/server/next-server.ts

```diff
@@ -47,6 +47,7 @@ export default class Server {
   renderOpts: {
     poweredByHeader: boolean
     ampBindInitData: boolean
+    buildWatcherStyle: string
```

Comment:
I don't think this should be an option 🤔

Reply:
Being, I think there should be one style and it shouldn't be opt-in or opt-out. Instead it should cover all cases 👍

Reply:
Updated 👍

Reply:
Yeah, I was undecided about this one because it didn't make much sense to have such type of config inside a project-related configuration file.